tests/zone.test.ts

```diff
@@ -0,0 +1,119 @@
+import { getDefaultAddress, getAddresses, addAddress, removeAddress, removeCoin, setAddresses } from "../src/zone";
+import { parseZoneFile, makeZoneFile } from "../src/zone";
+
+const file1 = `$ORIGIN muneeb.id
+$TTL 3600
+_http._tcp URI 10 1 "https://gaia.blockstack.org/hub/1J3PUxY5uDShUnHRrMyU6yKtoHEUPhKULs/0/profile.json"
+`
+const file2 = `$ORIGIN muneeb.id
+$TTL 3600
+_http._tcp URI 10 1 "https://gaia.blockstack.org/hub/1J3PUxY5uDShUnHRrMyU6yKtoHEUPhKULs/0/profile.json"
+_btc._addr.muneeb.id. IN TXT "0x123"
+_stx._addr.muneeb.id. IN TXT "0xABC"
+`
+const zoneFileJson1 = parseZoneFile(file1);
+const zoneFileJson2 = parseZoneFile(file2);
+
+describe('zone file without txt', () => {
+    test('initial state', () => {
+        let testJson = parseZoneFile(file1);
+
+        expect(testJson.$origin).toEqual('muneeb.id');
+        expect(testJson.txt).toEqual(undefined);
+        expect(getDefaultAddress(testJson, 'btc')).toEqual(null);
+        expect(getAddresses(testJson, 'stx')).toEqual([]);
+    });
+    test('add and remove address', () => {
+        let testJson = parseZoneFile(file1);
+
+        addAddress(testJson, 'btc', '1234', true);
```

Comment:
I think it would be better to have real addresses used in these tests

Reply:
maybe i can put shortened version of real addresses, as the reason I used fake ones was to make it easier to read / verify that the tests are written correctly

Reply:
I think full addresses are important because it also tests for any potential truncation or formatting related issues.